iocore/net/Connection.cc

```diff
@@ -212,6 +213,17 @@ Server::setup_fd_for_listen(bool non_blocking, const NetProcessor::AcceptOptions
   if ((res = safe_setsockopt(fd, SOL_SOCKET, SO_REUSEADDR, SOCKOPT_ON, sizeof(int))) < 0) {
     goto Lerror;
   }
+  REC_ReadConfigInteger(listen_per_thread, "proxy.config.exec_thread.listen");
```

Comment:
@vmamidi It might be better to have `listen_per_thread` as a member of AcceptOptions.

Reply:
I am not so sure as listening on all threads is not an Accept Option though we do it using socket options.